cmd/pgo-scheduler/scheduler/pgbackrest.go

```diff
@@ -136,7 +136,7 @@ func (b BackRestBackupJob) Run() {
 			"selector":  selector,
 			"error":     err,
 			"podsFound": len(pods.Items),
-		}).Error("pods returned does not equal 1, it should")
+		}).Error("error getting pods from selector: 0 or more then 1 pods present")
```

Comment:
```suggestion
		}).Error("error getting pods from selector: 0 or more than 1 pods present")
```

Reply:
@nonemax please fix it and check all messages too

Reply:
fixed https://github.com/percona/percona-postgresql-operator/pull/45/commits/f1cc9b16e769c3f45b077f836a339f1be7bf1f9f